frontend/src/component/feature/FeatureStrategy/FeatureStrategyMenu/FeatureStrategyMenuCards/FeatureStrategyMenuCards.tsx

```diff
@@ -102,113 +133,140 @@ export const FeatureStrategyMenuCards = ({
                     </IconButton>
                 )}
             </TitleRow>
-            {allStrategies ? (
-                <>
-                    <StyledTypography color='textSecondary'>
-                        Default strategy for {environmentId} environment
-                    </StyledTypography>
-                    <GridSection>
-                        <CardWrapper key={defaultStrategy.name}>
-                            <FeatureStrategyMenuCard
-                                projectId={projectId}
-                                featureId={featureId}
-                                environmentId={environmentId}
-                                strategy={defaultStrategy}
-                                defaultStrategy={true}
-                            />
-                        </CardWrapper>
-                    </GridSection>
-                </>
-            ) : null}
-            <ConditionallyRender
-                condition={templates.length > 0}
-                show={
+            <ScrollableContent>
```

Comment:
These big chunks is just moving some components around.

Reply:
Oh, interesting. So you're creating a separate scroll zone for strategies? Shouldn't release templates also get that treatment? Or .. what happens if you have 100 of each? (more realistically 20, but you know)

Reply:
Actually the `FeatureReleasePlanCard` live inside `FeatureStrategyMenuCards`. There is no `FeatureReleasePlanCards`.